YGOProAnalyticsServer/EventHandlers/YgoProAnalysisBasedOnDataFromYgoProServer.cs

```diff
@@ -234,8 +234,6 @@ private List<Decklist> _removeDuplicatesAndMerge(List<Decklist> decklists1, List
         private bool isBanlistOk(DuelLog duelLog)
         {
             return _duelLogNameAnalyzer.IsAnyBanlist(duelLog.Name)
```

Comment:
update docs for IsAnyBanlist() (info about internal IsAnyBanlist implementation required if you want remove IsDuelVersusAI and IsNoDeckCheckEnabled checks here)